include/fill_n.hpp

```diff
@@ -0,0 +1,43 @@
+// ================================ FILL N ================================== //
+// Project: The Experimental Bit Algorithms Library
+// Name: fill_n.hpp
+// Description: bit_iterator overloads for std::fill_n
+// Creator: Vincent Reverdy
+// Contributor(s): Vincent Reverdy [2019]
+//                 Collin Gress [2019]
+// License: BSD 3-Clause License
+// ========================================================================== //
+#ifndef _FILL_N_HPP_INCLUDED
+#define _FILL_N_HPP_INCLUDED
+// ========================================================================== //
+
+// ============================== PREAMBLE ================================== //
+// C++ standard library
+// Project sources
+// Third-party libraries
+
+namespace bit {
+
+// TODO
+template <class OutputIt, class Size>
+constexpr bit_iterator<OutputIt> fill_n(bit_iterator<OutputIt> first, Size count, 
+    const bit_value& bv) {
+  bit_iterator<OutputIt> last = first + count;
```

Comment:
Would you mind explaining why this is possible? Otherwise we can leave the discussion open just until we meet tomorrow, if you don't mind, so that  I can ask about it then.